projects/plugins/crm/includes/ZeroBSCRM.DAL3.Helpers.php

```diff
@@ -708,7 +708,7 @@ function zeroBS_getDemoCustomer(){
 
 	$demoData = array(
 
-		'status' => array('Lead','Customer'),
+		'status'           => array( 'Lead', 'Customer' ),
```

Comment:
This one is used by the Mail Campaigns extension. I took the liberty to revert it back to Customer.